radicle/src/git.rs

```diff
@@ -39,6 +39,13 @@ pub enum ListRefsError {
     InvalidRef(#[from] RefError),
 }
 
+impl From<&RemoteId> for RefString {
```

Comment:
I think it might make more sense to use `Component` here, similar to this https://github.com/radicle-dev/radicle-link/blob/master/link-crypto/src/peer.rs#L205-L213